tdrs-frontend/src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { Footer, Logo, SocialLinks } from '@trussworks/react-uswds'
+
+import ACFLogo from '../../assets/placeholder.png'
```

Comment:
See @cmajel 's comment about adding in the logo.

Reply:
This was addressed in the frontend remediations PR currently up for review in the raft-tech repo.

https://github.com/raft-tech/TANF-app/pull/262